src/SparkLendFreezerMom.sol

```diff
@@ -66,7 +66,7 @@ contract SparkLendFreezerMom is ISparkLendFreezerMom {
         emit Rely(usr);
     }
 
-    function deny(address usr) external override onlyOwner {
+    function deny(address usr) external override auth {
```

Comment:
Multisig can now deny another ward, does this present any issues?

Reply:
Can't think of any really just wanted to bring it up

Reply:
I've added a bigger readme section about this.